### ISSUE TITLE
Create your first PR

### ISSUE THREAD

**Issue by mkamranhamid** [mkamranhamid/my-first-hacktoberfest#18]
Create your first PR by following the steps described in [Readme](https://github.com/mkamranhamid/my-first-hacktoberfest/blob/master/README.md) file

**Note**: _This PR is only for developers who are beginners or juniors or someone who hasn't made a pull request yet. If you think you are experienced more than a beginner or junior level then I would recommend clicking [this](https://github.com/search?p=4&q=label%3Ahacktoberfest+state%3Aopen+no%3Aassignee+is%3Aissue&type=Issues) link to find all the open issues with hacktoberfest label_

**Comment by abdullah2205** [comment]
Hi I am new to github, can I contribute to this project?

**Comment by mkamranhamid** [comment]
Yes, this repository is for learning `git` commands and features so if you're new please feel free to contribute
